server/cmt_cmds.go

```diff
@@ -249,3 +258,96 @@ $ %s query block --%s=%s <hash>
 
 	return cmd
 }
+
+func BootstrapStateCmd(appCreator types.AppCreator) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bootstrap-state",
+		Short: "Bootstrap CometBFT state at an arbitrary block height using a light client",
+		Long: `
```

Comment:
If it is going to say the same thing, why add a long description? I think we should remove it.